util/topgen/lib.py

```diff
@@ -333,24 +333,22 @@ def get_unused_resets(top):
 def is_templated(module):
     """Returns an indication where a particular module is templated
     """
-    if "attr" not in module:
-        return False
-    elif module["attr"] in ["templated"]:
-        return True
-    else:
-        return False
+    return module.get('attr') in ["templated"]
```

Comment:
Nit: If re-pushing, this could be
```suggestion
    return module.get('attr') == "templated"
```
but no big deal if you're happy to merge without.

Reply:
o yeah i just used `in` just in case in the future we needed to include more things.